lib/Traits/traitsClassToModelClass.ts

```diff
@@ -1,11 +1,11 @@
-import { createTransformer } from "mobx-utils";
+import memoize from "lodash-es/memoize";
 import CreateModel from "../Models/CreateModel";
 import ModelTraits from "./ModelTraits";
 import TraitsConstructor from "./TraitsConstructor";
 
-const traitsClassToModelClass = createTransformer(function<
-  T extends ModelTraits
->(traitsClass: TraitsConstructor<T>) {
+const traitsClassToModelClass = memoize(function<T extends ModelTraits>(
```

Comment:
Could you please add a version of your comment from the PR as a comment in the code here? It's useful context for understanding why we're using memoize here.

Reply:
Good point! I have added a note.